src/sentry/tasks/post_process.py

```diff
@@ -480,6 +480,29 @@ def should_update_escalating_metrics(event: Event) -> bool:
     )
 
 
+@instrumented_task(
+    name="sentry.issues.tasks.post_process.post_process_group",
```

Comment:
During the switch over the task name will change to this. We can either retain this name going forward, or I can have the long term name stay as `sentry.tasks.post_process.post_process_group`. I'm open to any alternative names as well.

Reply:
i think it's fine to change the name -- we may want to move this task into the `issues` module anyways. we'll have to ensure that any alerts/metrics which use the current task name are changed over, and during the switch we may have false alarms, so mind posting in #discuss-issues whenver this goes out? 🙏🏼